packages/elements/src/react/hooks/use-third-party-provider.hook.ts

```diff
@@ -40,16 +47,18 @@ export const useThirdPartyProvider = <
   TActor extends ActorRef<any, SignInRouterEvents> | ActorRef<any, SignUpRouterEvents>,
 >(
   ref: TActor,
-  provider: OAuthProvider | Web3Provider | SamlStrategy,
+  provider: OAuthProvider | Web3Provider | SamlStrategy | EnterpriseSSOStrategy,
 ): UseThirdPartyProviderReturn => {
   const isProviderEnabled = useIsProviderEnabled(provider);
   const isSaml = isSamlStrategy(provider);
-  const details = isSaml
-    ? {
-        name: 'SAML',
-        strategy: 'saml' as SamlStrategy,
-      }
-    : providerToDisplayData[provider];
+  const isEnterpriseSSO = isEnterpriseSSOStrategy(provider);
+  const details =
+    isEnterpriseSSO || isSaml
+      ? {
+          name: 'SSO',
```

Comment:
Where this `name` is surfaced? On the `alt` of `Clerk.Icon`? 

Also, `saml` and `enterprise_sso` don't have predefined icons, since they depend on the enterprise connection type and FAPI doesn't expose this beforehand.

Reply:
@LauraBeatris Yes, it's largely surfaced there. Given the nature of Elements, it's just a default, and devs would know what connections they have available and either slot in their icon to Clerk.Icon or explicitly use their own.